dotCMS/src/main/java/com/dotcms/publishing/Publisher.java

```diff
@@ -225,7 +227,12 @@ protected boolean shouldProcess ( File assetFile ) throws IOException, DotPublis
         return true;
     }
 
-	public Set<String> getProtocols(){
+    private boolean doesPathContainLanguageId(File assetFile) {
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Parameter 'assetFile' is not assigned and could be declared final](https://app.codacy.com/manual/dotCMS/core/pullRequest?prid=5303548)